lib/ansible/modules/web_infrastructure/rundeck_acl_policy.py

```diff
@@ -0,0 +1,239 @@
+        return resp
+
+    def create_or_update_acl(self):
+        facts = self.get_acl()
+        if facts is None:
+            # If in check mode don't create project, simulate a fake project creation
+            if self.module.check_mode:
+                self.module.exit_json(changed=True, before={}, after=self.module.params["policy"])
+
+            _, info = self.request_rundeck_api("system/acl/%s.aclpolicy" % self.module.params["name"],
+                                               method="POST",
+                                               data={"contents": self.module.params["policy"]})
+
+            if info["status"] == 201:
+                self.module.exit_json(changed=True, before={}, after=self.get_acl())
+            elif info["status"] == 400:
+                self.module.fail_json(msg="Unable to validate acl %s. Please ensure it's a valid ACL" %
+                                          self.module.params["name"])
+            elif info["status"] == 409:
+                self.module.fail_json(msg="ACL %s already exists" % self.module.params["name"])
+            else:
+                self.module.fail_json(msg="Unhandled HTTP status %d, please report the bug" % info["status"],
+                                      before={}, after=self.get_acl())
+        else:
+            if facts["contents"] == self.module.params["policy"]:
+                self.module.exit_json(changed=False, before=facts, after=facts)
+
+            if self.module.check_mode:
+                self.module.exit_json(changed=True, before=facts, after=facts)
+
+            _, info = self.request_rundeck_api("system/acl/%s.aclpolicy" % self.module.params["name"],
+                                               method="PUT",
+                                               data={"contents": self.module.params["policy"]})
+
+            if info["status"] == 200:
+                self.module.exit_json(changed=True, before=facts, after=self.get_acl())
+            elif info["status"] == 400:
+                self.module.fail_json(msg="Unable to validate acl %s. Please ensure it's a valid ACL" %
+                                          self.module.params["name"])
+            elif info["status"] == 404:
+                self.module.fail_json(msg="ACL %s doesn't exists. Cannot update." % self.module.params["name"])
+
+    def remove_acl(self):
+        facts = self.get_acl()
+        if facts is None:
+            self.module.exit_json(changed=False, before={}, after={})
+        else:
+            # If not in check mode, remove the project
+            if not self.module.check_mode:
+                self.request_rundeck_api("system/acl/%s.aclpolicy" % self.module.params["name"], method="DELETE")
+            self.module.exit_json(changed=True, before=facts, after={})
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            state=dict(required=True, type='str', choices=['present', 'absent']),
```

Comment:
it is common to have the default `state=present`. Consider it for consistency reasons across modules. But it is not a blocker.